### ISSUE TITLE
Document provisioning backends

### ISSUE THREAD

**Issue by strugee** [NixOS/nixops#1473]
In looking at https://nixops.readthedocs.io/en/latest/, I can't find a list of supported provisioning backends. Seems like this should be listed somewhere? Maybe I'm just missing something, but I searched for a few backend names as well as the string `backend` and couldn't find anything.

**Comment by Mic92** [comment]
Yes. There are not documented and it seems also not functional anymore at least the memory backend I tried.
